docs/source/develop.rst

```diff
@@ -78,28 +98,42 @@ The test suite contains three kinds of tests
     These are rare and mostly for testing the command line interface.
 
 If you are comfortable with the Tornado interface then you will be happiest
-using the ``@gen_cluster`` style of test
+using the ``@gen_cluster`` style of test, e.g.
 
 .. code-block:: python
 
-   from distributed.utils_test import gen_cluster
+    # tests/test_submit.py
 
-   @gen_cluster(client=True)
-   def test_submit(c, s, a, b):
-       assert isinstance(c, Client)
-       assert isinstance(s, Scheduler)
-       assert isinstance(a, Worker)
-       assert isinstance(b, Worker)
+    from distributed.utils_test import gen_cluster
```

Comment:
Can you condense these imports to import multiple things per line. Something like:

```python
from distributed.utils_test import gen_cluster, inc
from distributed import Client, Scheduler, Worker, Future
```